### ISSUE TITLE
animated_vector 0.2.1 fails to compile

### ISSUE THREAD

**Issue by adil192** [HrX03/animated_vector#5]
Looks like it's using some removed deprecated Color methods and fails to compile. Interestingly, animated_vector 0.2.0 is fine. My guess would be that you're using an old version of Flutter?

My logs are spammed with this:
```
../../../.pub-cache/hosted/pub.dev/animated_vector-0.2.1/lib/src/data.dart:233:35: Error: The method 'withValues' isn't defined for the class 'Color'.
 - 'Color' is from 'dart:ui'.
Try correcting the name to the name of an existing method, or defining a method named 'withValues'.
          const Color(0xFFFFFFFF).withValues(alpha: evaluated.alpha),
                                  ^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/animated_vector-0.2.1/lib/src/data.dart:521:33: Error: The method 'withValues' isn't defined for the class 'Color'.
 - 'Color' is from 'dart:ui'.
Try correcting the name to the name of an existing method, or defining a method named 'withValues'.
          ..color = strokeColor.withValues(
                                ^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/animated_vector-0.2.1/lib/src/data.dart:522:32: Error: The getter 'a' isn't defined for the class 'Color'.
 - 'Color' is from 'dart:ui'.
Try correcting the name to the name of an existing getter, or defining a getter or field named 'a'.
            alpha: strokeColor.a * evaluated.strokeAlpha!,
                               ^
../../../.pub-cache/hosted/pub.dev/animated_vector-0.2.1/lib/src/data.dart:535:23: Error: The method 'withValues' isn't defined for the class 'Color'.
 - 'Color' is from 'dart:ui'.
Try correcting the name to the name of an existing method, or defining a method named 'withValues'.
            fillColor.withValues(alpha: fillColor.a * evaluated.fillAlpha!),
                      ^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/animated_vector-0.2.1/lib/src/data.dart:535:51: Error: The getter 'a' isn't defined for the class 'Color'.
 - 'Color' is from 'dart:ui'.
Try correcting the name to the name of an existing getter, or defining a getter or field named 'a'.
            fillColor.withValues(alpha: fillColor.a * evaluated.fillAlpha!),
                                                  ^
../../../.pub-cache/hosted/pub.dev/animated_vector-0.2.1/lib/src/shapeshifter.dart:769:13: Error: The getter 'a' isn't defined for the class 'Color'.
 - 'Color' is from 'dart:ui'.
Try correcting the name to the name of an existing getter, or defining a getter or field named 'a'.
  if (color.a == 0xFF) return "#${radixString.substring(2)}";
            ^
```

These errors are also recorded on the [pub.dev score page](https://pub.dev/packages/animated_vector/score).

**Comment by KodeBarista** [comment]
Same for building Yaru app for Linux

**Comment by dvd-bnc** [comment]
hi sorry for the mess! i was using methods from the next flutter stable version and as such they are not available yet in the current stable branch, i guess 0.2.2 should fix it, do give it a try if possible and let me know if it works now! sorry again for the annoyance

**Comment by KodeBarista** [comment]
Issue seems to have been fixed with `0.2.2`

**Comment by adil192** [comment]
Thank you!
